layers/+lang/emacs-lisp/README.org

```diff
@@ -93,6 +97,15 @@ function or press ~o~ to go out of it.
 
 7) Press ~a~ to stop debugging.
 
+* Nameless
+Nameless hides package namespaces in your emacs-lisp code, and replaces it by leading ~:~
+It can be toggled by ~SPC m :~.
+
+To have it automatically on, you need to define a layer variable:
+~(emacs-lisp :variables emacs-lisp-nameless-mode t)~ in ~.spacemacs~ ~dotspacemacs/layers~
```

Comment:
Spacemacs Document convention: `~` is used for <kbd>keybindings</kbd> and `=` is used for normal `code`.

Reply:
my bad

Reply:
@nixmaniack solved